ts/components/screens/AnimatedScreenContent.tsx

```diff
@@ -133,6 +133,11 @@ export default class AnimatedScreenContent extends React.Component<
         </Animated.ScrollView>
 
         <Animated.View
+          onLayout={(event: LayoutChangeEvent) =>
```

Comment:
Please check what we said in the sprint review.

Reply:
I introduced the control at line 134 to avoid status is refreshed. As hypothesized, onLayout is called also after the first time the component is rendered.